test/include/materials/LinearNonLinearIterationInterfaceMaterial.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InterfaceMaterial.h"
+
+// Forward Declarations
+class LinearNonLinearIterationInterfaceMaterial;
+
+template <>
+InputParameters validParams<LinearNonLinearIterationInterfaceMaterial>();
```

Comment:
legacy

Reply:
I tried removing the legacy line, it compiles but it errors out on execution complaining about undefined parameters

Reply:
Can you share the exact error?

Reply:
Do we still need this one?